library/src/scala/quoted/Quotes.scala

```diff
@@ -2812,6 +2812,9 @@ trait Quotes { self: runtime.QuoteUnpickler & runtime.QuoteMatching =>
 
     /** Methods of the module object `val AppliedType` */
     trait AppliedTypeModule { this: AppliedType.type =>
+      /** Applied the type constructor `T` to a list of type arguments `T_1,..,T_n` to create `T[T_1,..,T_n]` */
+      @experimental
```

Comment:
Why is this considered experimental?

Reply:
all new API must be experimental as adding an API without a new minor version bump does not conform to our compatibility policy

Reply:
This will be set to stable for 3.2.0. I prefer to do a single PR that removes all `@experimental` at the same time.

Reply:
Makes sense, thanks for explaining!